src/com/ss/ita/kata/Eight.java

```diff
@@ -5,7 +5,6 @@
      * @param time
      * @return
      */
-    int Liters(double time);
```

Comment:
We must have two methods, liters and Liters

Reply:
@Khrystyna-Pelekh, I understand that we will have some conflict in the interface implementation, but actually, this method is useless and we all can remove it from our project.